hudi-client/hudi-client-common/src/main/java/org/apache/hudi/index/hbase/AbstractHoodieHBaseIndex.java

```diff
@@ -0,0 +1,288 @@
+package org.apache.hudi.index.hbase;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.HBaseConfiguration;
+import org.apache.hadoop.hbase.HRegionLocation;
+import org.apache.hadoop.hbase.TableName;
+import org.apache.hadoop.hbase.client.BufferedMutator;
+import org.apache.hadoop.hbase.client.Connection;
+import org.apache.hadoop.hbase.client.ConnectionFactory;
+import org.apache.hadoop.hbase.client.Get;
+import org.apache.hadoop.hbase.client.HTable;
+import org.apache.hadoop.hbase.client.Mutation;
+import org.apache.hadoop.hbase.client.RegionLocator;
+import org.apache.hadoop.hbase.client.Result;
+import org.apache.hadoop.hbase.util.Bytes;
+import org.apache.hudi.common.HoodieEngineContext;
+import org.apache.hudi.common.model.HoodieRecordPayload;
+import org.apache.hudi.common.table.HoodieTableMetaClient;
+import org.apache.hudi.common.table.timeline.HoodieInstant;
+import org.apache.hudi.common.table.timeline.HoodieTimeline;
+import org.apache.hudi.common.util.ReflectionUtils;
+import org.apache.hudi.config.HoodieHBaseIndexConfig;
+import org.apache.hudi.config.HoodieWriteConfig;
+import org.apache.hudi.exception.HoodieDependentSystemUnavailableException;
+import org.apache.hudi.index.AbstractHoodieIndex;
+import org.apache.hudi.table.AbstractHoodieTable;
+import org.apache.log4j.LogManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * @author xianghu.wang
+ * @time 2020/6/11
+ * @description
+ */
```

Comment:
ditto

Reply:
@wangxianghu another minor thing.. we generally don't do `@author` and other headers in source (we have git blame already) .. so may be revert that in all the files as well?

Reply:
> @wangxianghu another minor thing.. we generally don't do `@author` and other headers in source (we have git blame already) .. so may be revert that in all the files as well?

@vinothchandar, yes, it was generated by idea automatically and I have deleted it.